specs/wp-manage-domains-spec.js

```diff
@@ -185,4 +187,84 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
 			return await shoppingCartWidgetComponent.empty();
 		} );
 	} );
+
+	describe( 'Transfer a domain to an existing site (partial) @parallel', function() {
+		const domain = 'andrija.me';
```

Comment:
I think this should probably be put in to the encrypted config file

Reply:
This is just a random taken domain. Nothing important in any way. :)